src/main/resources/templates/fragments.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="https://www.thymeleaf.org/">
+<head th:fragment="head">
+    <meta charset="UTF-8"/>
+    <title>Recipes</title>
```

Comment:
Do we want the title element to have a hard coded value or should it also have a th:text attribute like the index.html's header originally had?